src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        SimpleDateFormat dateFormat = new SimpleDateFormat("dd.MM.yyyy");
+        StringBuilder report = new StringBuilder();
+        String newline = System.lineSeparator();
```

Comment:
it's not neccesary to create variable for that, but if you want to do that, please make it final and capitalize to mark it as constant